### DISCUSSION TITLE
Question about the title language

### DISCUSSION THREAD

**Discussion by EchterAlsFake** [EchterAlsFake/unofficial-api-for-pornhub#2] [Q&A] [answered]
Hey, 

I have a question about the titles. 

If I call an Object like:

client = phub.Client()
video = client.get("some url here")

title = video.title

then the title is sometimes in french. I guess this is because some videos are translated in multiple languages. I wanted to ask, if it is possible to specify English as the language for fetching the title.

**Comment by Egsagon** [comment]
Hello.
Thanks for pointing this out because it is entirely my bad. i forgot about video languages.
As of now, you can set the video language by putting this line after importing phub.
```python
phub.consts.HEADERS['Accept-Language'] = 'en,en-US'
```
This will override the default requests language value (`fr,fr-FR;q=0.8,en-US;q=0.5,en;q=0.3` by default because i'm french).
I'll add a feature to set language per video when i'll find time.

**Reply by EchterAlsFake** [reply]
Thank you very much :)

**Reply by Egsagon** [reply, marked as answer]
Implemented with 63ef9d2a1c1946cb3554880b98316d9dc2cabc11 :)
The default language is now english but you can change that while initialising the client:
```python
client = phub.Client(language = 'fr') # or 'es', 'en', etc.
```
All objects created with this client will inherit that language.
